src/package/nuspec/TestPlatform.Extensions.TrxLogger.nuspec

```diff
@@ -24,13 +24,12 @@
         <dependency id="Newtonsoft.Json" version="$JsonNetVersion$"/>
         <dependency id="Microsoft.TestPlatform.ObjectModel" version="$Version$"/>
       </group>
-      <group targetFramework="netstandard1.5">
-        <dependency id="System.Collections.NonGeneric" version="[4.0.1, )" />
-        <dependency id="System.Security.Principal.Windows" version="[4.0.1, )" />
-        <dependency id="System.Collections.Specialized" version="[4.0.1, )" />
+      <group targetFramework="netstandard2.0">
+        <!-- REVIEW ME: Why don't we have dependency to Newtonsoft.Json here?  -->
```

Comment:
Other TFMs are declaring dependency on `Newtonsoft.Json`. Why are we not doing it for `netstandard2.0`?

Reply:
That looks like an error.

Reply:
I will fix this one as a follow-up PR so things don't get mixed up.